covid19-etl/utils/metadata_helper.py

```diff
@@ -98,8 +100,14 @@ def get_str_latest_submitted_date_idph(self):
         }"""
         variables = {"filter": {"=": {"project_id": self.project_id}}}
         query_res = self.query_guppy(query_string, variables)
-        loc = query_res["data"]["location"][0]
-        return loc["date"]
+        if "data" not in query_res or "location" not in query_res["data"]:
+            raise Exception(
+                "Did not receive any data from Guppy. Is the token expired?"
+            )
+        loc = query_res["data"]["location"]
+        if (len(loc)) > 0:
+            return loc["date"]
```

Comment:
```suggestion
            return loc[0]["date"]
```

Reply:
Aaah! I should be extra careful with the changes that I make in the last moment! Good catch! ☮️  🙇